src/rard/research/models/work.py

```diff
@@ -49,6 +52,17 @@ class Meta:
 
     unknown = models.BooleanField(default=False)
 
+    introduction = models.OneToOneField(
+        "TextObjectField",
+        on_delete=models.SET_NULL,
+        null=True,
+        related_name="introduction_for_%(class)s",
+    )
+    plain_introduction = models.TextField(blank=False, default="")
+
+    def book_set(self):
+        return super().book_set().order_by("unknown", "order", "number")
+
```

Comment:
I don't understand what this is for. The meta class on the Book model already orders books by unknown, order, number... so I don't think this actually does anything.

Reply:
Ah, this shouldn't be here. This branch was created off the definite/possible before it was merged and before this had been deleted. Will delete, thank you for spotting it